libbindgen/tests/expectations/tests/namespace.rs

```diff
@@ -66,6 +66,7 @@ pub mod root {
             #[link_name = "_ZN1w3fooEv"]
             pub fn foo() -> root::C<::std::os::raw::c_int>;
         }
+        pub enum C { }
```

Comment:
I don't know why this is appearing here, even though the struct has already been declared a few lines above.